### DISCUSSION TITLE
How to use Camunda Modeler / BPMN embed?

### DISCUSSION THREAD

**Discussion by saschabrockel** [outline/outline#8160] [General]
I'm selfhosting Outline and I'm confused how to use the embed of Camunda Modeler. It appears and then probably wants to have a link but even if I give a URL like https://raw.githubusercontent.com/bpmn-io/bpmn-js-examples/refs/heads/main/url-viewer/resources/pizza-collaboration.bpmn which is an BPMN diagram, does not work.

Does anybody use it? There is also no information about it.

**Comment by tommoor** [comment]
Only models at `modeler.cloud.camunda.io` are supported for this integration- you embed them by pasting a shared url

**Reply by saschabrockel** [reply]
Thank you for the answer. But it is not documented yet or did I miss something? I was curious about what was working and tried several integrations like Camunda Modeler or Google Maps but was not successful.
